px-view-header.html

```diff
@@ -40,23 +40,23 @@
 <dom-module id="px-view-header">
   <template>
     <style include="px-view-header-styles"></style>
-    
-    <div id="container" class="header flex flex--justify">
-      <div id="left" class="flex flex--col flex--center">
+
+    <div id="container" class="header">
+      <div id="left">
         <content select=".left"></content>
```

Comment:
It is correct -- it's grabbing the content in the demo page with the `.left` class. But that class was actually superfluous, so I removed it and gave it an id `#left` instead. (Same with `.right`.)

See here for more detail:
https://stackoverflow.com/questions/28318520/polymer-template-what-are-valid-selects-for-content-insertion-points